lib/ledger_sync/resources/bill_payment.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+require_relative 'account'
+require_relative 'currency'
+require_relative 'department'
+require_relative 'vendor'
+require_relative 'bill_payment_line_item'
+
+module LedgerSync
+  class BillPayment < LedgerSync::Resource
```

Comment:
I don't know why I'm just now thinking of this, but is there a reason we wouldn't just use `Payment`?  The attributes look the same some attributes.  Is `Payment` one direction and this is the other?

Reply:
@mattgmarcus I'm curious if you have thoughts on this.

On one hand it reminds me of bi-direction POs in MT.

On the other hand, it also seems similar to our current counterparty issue of having one resource that could be a vendor, customer, or employee.  Perhaps specificity at the resource class level is best.

Reply:
I think the relationship is more like `Invoice` -> `Payment`, `Bill` -> `BillPayment`